spring-cloud-dataflow-server-core/src/main/resources/schema-hsqldb.sql

```diff
@@ -1,6 +1,15 @@
 
-CREATE TABLE TASK_DEFINITION  (
+CREATE TABLE TASK_DEFINITIONS  (
 	DEFINITION_NAME VARCHAR(255) NOT NULL PRIMARY KEY,
 	DEFINITION CLOB DEFAULT NULL
 );
 
+CREATE TABLE STREAM_DEFINITIONS  (
+	DEFINITION_NAME VARCHAR(255) NOT NULL PRIMARY KEY,
+	DEFINITION CLOB DEFAULT NULL
+);
+
+CREATE TABLE DEPLOYMENT_IDS  (
+	DEPLOYMENT_KEY VARCHAR(255) NOT NULL PRIMARY KEY,
+	DEPLOYMENT_ID VARCHAR(255) NOT NULL
+);
```

Comment:
How can we re-launch a task? I get a duplicate key error launching the task a second time:

```
Command failed org.springframework.cloud.dataflow.rest.client.DataFlowClientException: PreparedStatementCallback; SQL [INSERT into DEPLOYMENT_IDS (DEPLOYMENT_KEY, DEPLOYMENT_ID)values (?, ?)]; Duplicate entry 'timestamp.test3' for key 'PRIMARY'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'timestamp.test3' for key 'PRIMARY'
```